Update docs for grammar and clarity (#11)
MatX-inc/seqax: README.md

   NOTE: the `requirements-cpu.txt` is configured for CPU-based installation. For GPU or TPU installation, you may need a different install of JAX and jaxlib. See the [JAX install documentation](https://jax.readthedocs.io/en/latest/installation.html). If your GPU environment has a Torch-GPU installation, you may need to switch it to a Torch-CPU installation to avoid conflicts with JAX-GPU.

### Run on CPU for local development

For development and tests, you can run seqax on the CPU. Typically, you'd use our synthetic dataset (which is [checked into this repository](/synthetic_dataset.zarr)) or the [Huggingface data loader](#data-loaders), and you'd set XLA flags to simulate multiple devices to test that parallelism is working as intended:

```
XLA_FLAGS=--xla_force_host_platform_device_count=8 python -m train --config-name=local_test_synthetic +paths.model_name=synthetic_000
```

The `paths.model_name` flag specifies which subdirectory on disk within `/tmp` used to write model checkpoints. You'll typically want to change this when starting a new model run.

### Run on GPUs

> We have configured a range of model sizes to be trained on the C4 dataset with the Llama tokenizer. Browse the `configs/` directory to select your preferred configuration file. Each configuration file lists how to run it at the top.

Author: ```suggestion
We have configured a range of model sizes to be trained on the C4 dataset with the Llama tokenizer. Browse the `configs/` directory to select your preferred configuration file. Each configuration file lists how to run it.
```

> You typically want to set `paths.model_name` to a unique name for each distinct training run. This path specifies which subdirectory on disk are used to write model checkpoints.

Author: ```suggestion
You typically want to set `paths.model_name` to a unique name for each distinct training run. This path sepcifies the local subdirectory for model checkpoints.
```
